src/models/stocks.ts

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+
+import { users } from "./user";
+import type { IStock } from "@interfaces";
+import { mongoose } from "@services";
+import { STOCK_API_URL } from "@src/types/values";
+
+class Stocks extends mongoose.Repository<IStock> {
+	private static readonly stockApiClient = axios.create({
+		baseURL: STOCK_API_URL
+	});
+
+	constructor() {
+		super("Stock", {
+			server_id: {
+				type: String,
+				index: true,
+				required: true
+			},
+			user: {
+				type: String,
+				ref: users.model.modelName,
+				index: true,
+				required: true
+			},
+			symbol: {
+				type: String,
+				index: true
+			},
+			price: {
+				type: Number
+			},
+			currency: {
+				type: String
+			}
+		});
+	}
+
+	public async getPrice(symbol: string): Promise<IStock> {
+		try {
+			const { data } = await Stocks.stockApiClient.get(symbol);
+
+			const priceString: string = data
+				.split(`"${symbol}":{"sourceInterval"`)[1]
+				.split("regularMarketPrice")[1]
+				// eslint-disable-next-line prettier/prettier
+				.split("fmt\":\"")[1]
+				// eslint-disable-next-line prettier/prettier
+				.split("\"")[0];
+
+			const price = parseFloat(priceString.replace(",", ""));
+
+			const currencyMatch = data.match(/Currency in ([A-Za-z]{3})/);
+			const currency: string = currencyMatch ? currencyMatch[1] : null;
```

Comment:
same as above, no need to assert type

Reply:
Same deal here:

![image](https://user-images.githubusercontent.com/1410481/168925912-4b999dac-7dde-46b8-be1a-0df22523ab7b.png)

Reply:
Ok that's fine, if they're marked as `any` or `unknown` just add `as type` at the end of the line rather. More clear imo